packages/nx/src/hasher/hashing-impl.ts

```diff
@@ -5,7 +5,10 @@ export class HashingImpl {
   hashArray(input: string[]): string {
     const hasher = createHash('sha256');
     for (const part of input) {
-      hasher.update(part);
+      // intentional single equals to check for null and undefined
+      if (part != undefined) {
+        hasher.update(part);
+      }
```

Comment:
Right now if you pass `undefined` or `null` to `hasher.update` it would fail with:

```
The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView
```

I see no case for passing undefined. If we need a falsy value, we can always pass an empty string.